src/test/javascript/spec/component/lecture-unit/text-unit/text-unit.component.spec.ts

```diff
@@ -0,0 +1,117 @@
+import * as chai from 'chai';
+import * as sinonChai from 'sinon-chai';
+import * as sinon from 'sinon';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { TextUnitComponent } from 'app/overview/course-lectures/text-unit/text-unit.component';
+import { MockComponent, MockDirective, MockPipe, MockProvider } from 'ng-mocks';
+import { TranslatePipe } from '@ngx-translate/core';
+import { ArtemisMarkdownService } from 'app/shared/markdown.service';
+import { FaIconComponent } from '@fortawesome/angular-fontawesome';
+import { NgbCollapse, NgbTooltip } from '@ng-bootstrap/ng-bootstrap';
+import { ArtemisDatePipe } from 'app/shared/pipes/artemis-date.pipe';
+import { TextUnit } from 'app/entities/lecture-unit/textUnit.model';
+
+chai.use(sinonChai);
+const expect = chai.expect;
+describe('TextUnitFormComponent', () => {
+    const sandbox = sinon.createSandbox();
+    const exampleName = 'Test';
+    const exampleMarkdown = '# Sample Markdown';
+    const exampleHTML = '<h1>Sample Markdown</h1>';
+    let textUnitComponentFixture: ComponentFixture<TextUnitComponent>;
+    let textUnitComponent: TextUnitComponent;
+    let textUnit: TextUnit;
+
+    beforeEach(() => {
+        textUnit = new TextUnit();
+        textUnit.name = exampleName;
+        textUnit.content = exampleMarkdown;
+
+        TestBed.configureTestingModule({
+            imports: [],
+            declarations: [
+                TextUnitComponent,
+                MockComponent(FaIconComponent),
+                MockPipe(TranslatePipe),
+                MockDirective(NgbCollapse),
+                MockDirective(NgbTooltip),
+                MockPipe(ArtemisDatePipe),
+            ],
+            providers: [
+                MockProvider(ArtemisMarkdownService, {
+                    safeHtmlForMarkdown: (markdown: string) => exampleHTML,
+                    htmlForMarkdown: (markdown: string) => exampleHTML,
```

Comment:
GitHub Actions: `'markdown' is defined but never used`

Reply:
i mean i need to have the parameter to mock the method? Any idea how to do this? @FrankeLukas